docs/pages/blog/mui-x-end-v6-features.md

```diff
@@ -3,19 +3,19 @@ title: MUI X v6.18.0 and the latest features before the next major
 description: New components, polished features, better performance and more.
 date: 2023-11-13T00:00:00.000Z
 authors: ['josefreitas']
-card: false
+card: true
 tags: ['MUI X', 'News']
 ---
 
-<div style="max-width:680px;width: 100%; height: 170px; overflow: hidden; margin-bottom: 16px;">
+<div style="overflow: hidden; margin-bottom: 16px;">
   <a href="https://github.com/mui/mui-x/releases/tag/v6.18.0">
-    <img src="/static/blog/mui-x-end-v6-features/intro.png" alt="open v6.18.0 release page" style="width: 100%; height: 100%; object-fit: cover; object-position: center;" />
+    <img src="/static/blog/mui-x-end-v6-features/intro.png" alt="MUI X v6.18.0 release" width="1200" height="400" style="width: 100%; height: 100%; object-fit: cover; object-position: center;" />
```

Comment:
Is this a new standard size for intro cards?

`width="1200" height="400"`

Reply:
There's not a hard-enforced standard yet — just felt like the added 100px of height added a nice breathing room.